deploy/infra/modules/postgres.single.svc.bicep

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+@description('Server Name for Azure database for PostgreSQL')
+param serverName string
+
+@description('Database administrator login name')
+@minLength(5)
+param administratorLogin string
+
+@description('Database administrator password')
+@minLength(8)
+@secure()
+param administratorLoginPassword string
+
+@description('Azure database for PostgreSQL compute capacity in vCores (2,4,8,16,32)')
+param skuCapacity int = 2
+
+@description('Azure database for PostgreSQL sku name ')
+param skuName string = 'GP_Gen5_2'
+
+@description('Azure database for PostgreSQL Sku Size ')
+param skuSizeMB int = 51200
```

Comment:
This type should be string to pass sku parameter of 'Microsoft.DBforPostgreSQL/servers@2017-12-01'

Reply:
Changed to string.